src/ergw_prometheus.erl

```diff
@@ -105,6 +107,11 @@ declare() ->
 				  {labels, [name, type]},
 				  {buckets, [100, 300, 500, 750, 1000]},
 				  {help, "PFCP Request execution time."}]),
+
+    %% Termination cause metrics
+    prometheus_counter:declare([{name, termination_cause_total},
+                  {labels, [name, type]},
+                  {help, "Total number of termination causes"}]),
```

Comment:
@mgumz @fholzhauser shouldn't that include version and remote as tags as well? Maybe even direction and message type (for incoming requests)?

Reply:
Yes, we could consider adding those, however I'm not sure how it'd work consistently with the variety of sources these terminations can be initiated from. Proably it requires further discussion to work it out.